CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changelog
 
+## 3.0.0 (2022-09-xx)
+
+* Support for Symfony <5.4 dropped
+* Support for PHP < 8 dropped
+* remove deprecated functions getPayum, getHttpRequestVerifier and getTokenFactory in PayumController
+
 ## 2.5.0 (2022-07-xx)
 
-* Support for Symfony 5.0 - 5.3 dropped
+* Support for Symfony 4.0 - 4.3 dropped
```

Comment:
This change is not correct. In version 2.5, we dropped support for Symfony 5.0 - 5.3, only supporting Symfony 4.4, 5.4 and 6.0+

Reply:
Sorry. I will revert it